cache/cache.go

```
@@ -211,16 +213,12 @@ func (d *cacheInstance) GetValues(ctx context.Context, keys []string) ([]interfa
 		pipeResults = append(pipeResults, redisPipe.Get(ctx, k))
 	}
 
-	_, err = redisPipe.Exec(ctx)
-	if err != nil {
-		logger.Log.WithError(err).Error(groupError)
-		return nil, missingKeys, err
-	}
+	// no need to care about the error here since an error is thrown if one key is not found
+	cmdRes, _ := redisPipe.Exec(ctx)
```

What about execution error?

---

we shouldn't ignore any errors